values/mainnet/network/api-read-slave-1.yaml

```diff
@@ -12,7 +12,7 @@ init:
 image:
   repository: glif/lotus:v1.16.0
 
-Lotus:
+lotus:
```

Comment:
Are you sure we need to have this change?
I think we need to revert or make additional changes